CHANGELOG.txt

```diff
@@ -41,6 +41,7 @@ v27.0.00
 
     Bug Fixes
         System: fixed PHP 8+ compatibility in CustomFieldIDs migration file
+        Students: fixed view all option in report_students_byFormGroup 
```

Comment:
I tend to refer to files by their action name, rather than their filename, to help keep the changelog readable by the Gibbon layperson